api/utils/keys/yubikey.go

```diff
@@ -481,7 +539,52 @@ func (y *YubiKey) getPrivateKey(slot piv.Slot) (*PrivateKey, error) {
 		return nil, trace.Wrap(err)
 	}
 
-	return NewPrivateKey(priv, keyPEM)
+	key, err := NewPrivateKey(priv, keyPEM)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	return key, nil
+}
+
+// SetPin sets the YubiKey PIV PIN.
```

Comment:
Does this need some sort of user interaction?

Reply:
No, it just needs the correct old PIN, and only has 3 attempts before the card is locked.

Reply:
Ta. I'd say in the docs that it doesn't need user interaction, just to be extra clear.

Reply:
(By "docs" I meant godoc.)